research/deeplab/datasets/build_cityscapes_data.py

```diff
@@ -113,17 +113,23 @@ def _get_files(data, dataset_split):
 
   Args:
     data: String, desired data ('image' or 'label').
-    dataset_split: String, dataset split ('train', 'val', 'test')
+    dataset_split: String, dataset split ('train_fine', 'val_fine', 'test_fine')
 
   Returns:
     A list of sorted file names or None when getting label for
       test set.
   """
-  if data == 'label' and dataset_split == 'test':
+  if data == 'label' and dataset_split == 'test_fine':
     return None
+  if dataset_split == 'train_fine':
+    split_dir = 'train'
+  elif dataset_split == 'val_fine':
```

Comment:
Would you like to support the case, where data == 'image' and dataset_split == 'test_fine'?

Reply:
Updated.

Since there are no labels, the `train.py`/`eval.py` makes no sense, `vis.py` seems to work:

```
python deeplab/vis.py \
    --logtostderr \
    --vis_split="test_fine" \
    --model_variant="xception_65" \
    --atrous_rates=6 \
    --atrous_rates=12 \
    --atrous_rates=18 \
    --output_stride=16 \
    --decoder_output_stride=4 \
    --vis_crop_size="1025,2049" \
    --dataset="cityscapes" \
    --max_number_of_iterations=1 \
    --colormap_type="cityscapes" \
    --checkpoint_dir=${PATH_TO_CHECKPOINT} \
    --vis_logdir=${PATH_TO_VIS_DIR} \
    --dataset_dir=${PATH_TO_DATASET}
```

Example `leftImg8bit/test/berlin/berlin_000082_000019_leftImg8bit.png`:

![000082_image](https://user-images.githubusercontent.com/4346993/87586709-9eac6800-c6e9-11ea-9680-e9aca171030d.png)
![000082_prediction](https://user-images.githubusercontent.com/4346993/87586717-a23fef00-c6e9-11ea-8021-0bc97e14f236.png)